packages/mdc-slider/README.md

```diff
@@ -125,6 +144,31 @@ for the minimum and maximum values, which can always be set. This is to ensure c
 The step value can be any positive floating-point number, or `0`. When the step value is `0`, the
 slider is considered to not have any step.
 
+For discrete slider, it requires to have a step value other than `0`. So if a step is not provided,
```

Comment:
If you set a step to a negative value, it will throw you an error on your face :p
I will document that part~